src/foundry/common/documents.mjs/baseActor.d.ts

```diff
@@ -2,8 +2,11 @@ import { DocumentMetadata } from '../abstract/document.mjs';
 import { Document } from '../abstract/module.mjs';
 import { BaseActiveEffect } from './baseActiveEffect';
 import { BaseItem } from './baseItem';
+import * as data from '../data/data.mjs';
+
+export declare class BaseActor extends Document<data.ActorData, null> {
```

Comment:
the second generic parameter should probably be the configured `Token` class because TokenActors can be embedded in `Token`s, if I am not mistaken.

Reply:
I'm not entirely sure on that one. Should it be `Token | null` or just Token?

Reply:
Just Token ist fine, the `parent` property is typed as `Parent | null`

Reply:
I think I'll have to leave this as null or AnyDocument for now since Token needs to be typed first